src/components/inputs/time-input/time-input.js

```diff
@@ -148,8 +148,8 @@ export class TimeInput extends React.Component {
           onBlur={this.handleBlur}
           isAutofocussed={this.props.isAutofocussed}
           isDisabled={this.props.isDisabled}
-          hasError={this.props.hasError}
           isReadOnly={this.props.isReadOnly}
+          hasError={this.props.hasError}
```

Comment:
This was accidental, right? Better undo this to avoid the additional file being changed for nothing.

Reply:
hm? no I need `hasError` in the time-input-body

Reply:
This is the diff I'm seeing
![image](https://user-images.githubusercontent.com/2941328/57853993-69f00880-77e7-11e9-824c-c7596e86d344.png)

So it was already there, just moved 2 lines below? 😄

Reply:
oh